src/pages/ProductPage/Product.js

```diff
@@ -1,7 +1,289 @@
-import React from "react";
+import React, { useState, useEffect } from 'react';
+import ProductAll from './ProductAll';
+import ProductChocolate from './ProductChocolate';
+import ProductJelly from './ProductJelly';
+import ProductCandy from './ProductCandy';
+import ProductCookie from './ProductCookie';
+import ProductCake from './ProductCake';
+import Dropdown from './Dropdown';
+import './Product.scss';
 
 function Product() {
-  return <div></div>;
+  const [all, setAll] = useState([]);
+  const [chocolates, setChocolate] = useState([]);
+  const [jelly, setJelly] = useState([]);
+  const [candy, setCandy] = useState([]);
+  const [cookie, setCookie] = useState([]);
+  const [cake, setCake] = useState([]);
+  const [filter, setFilter] = useState([]);
+  const [activeIndex, setActiveIndex] = useState(0);
+  const [dropdownMenu, setDropDownMenu] = useState(false);
+
+  useEffect(() => {
+    fetch('/data/all.json')
+      .then(response => response.json())
+      .then(result => setAll(result));
+  }, []);
+
+  useEffect(() => {
+    fetch('/data/chocolate.json')
+      .then(response => response.json())
+      .then(result => setChocolate(result));
+  }, []);
+
+  useEffect(() => {
+    fetch('/data/candy.json')
+      .then(response => response.json())
+      .then(result => setCandy(result));
+  }, []);
+
+  useEffect(() => {
+    fetch('/data/jelly.json')
+      .then(response => response.json())
+      .then(result => setJelly(result));
+  }, []);
+
+  useEffect(() => {
+    fetch('/data/cookie.json')
+      .then(response => response.json())
+      .then(result => setCookie(result));
+  }, []);
+
+  useEffect(() => {
+    fetch('/data/cake.json')
+      .then(response => response.json())
+      .then(result => setCake(result));
+  }, []);
```

Comment:
필터링할 때마다, 현재 필터링한 옵션이 어떤 내용인지 state 하나로 관리하면 좋을 것 같습니다. 해당 state로 이 부분을 동적으로 요청하면 되겠네요! 간단한 형태 하나를 예시로 들어드리겠습니다.
목데이터를 활용한다면 다음 형태겠네요.
```suggestion
  useEffect(() => {
    fetch(`/data/${category}.json`)
      .then(response => response.json())
      .then(result => setProducts(result));
  }, []);
```

Reply:
도와주셔서 감사합니다. 코드 공부 다시 해보겠습니다..!!